### PR TITLE
[full-ci] ocisBump_6.6.0

### PR DESCRIPTION
**Pull request by ScharfViktor**
part of https://github.com/owncloud/ocis/issues/10345

### CODE REVIEW

File: tests/e2e/cucumber/steps/ui/resources.ts [hunk 1]
```diff
@@ -948,6 +948,7 @@ Then(
   ): Promise<void> {
     const { page } = this.actorsEnvironment.getActor({ key: stepUser })
     const resourceObject = new objects.applicationFiles.Resource({ page })
+    await page.pause()
```
Comment (AlexAndBear) [inline tests/e2e/cucumber/steps/ui/resources.ts:951]: Is the pause still necessary?

Comment (ScharfViktor) [reply tests/e2e/cucumber/steps/ui/resources.ts:951]: no. I'll revert it

**Review by AlexAndBear — APPROVED** [review]

**Comment by AlexAndBear** [comment]
Fuck I wasn't aware that the auto merge was enabled, I will approve your next pr, sorry 😔

File: tests/e2e/cucumber/steps/ui/resources.ts (hunk 1 above)
Comment (ScharfViktor) [reply tests/e2e/cucumber/steps/ui/resources.ts:951]: https://github.com/owncloud/web/pull/11809

**Comment by ScharfViktor** [comment]
> Fuck I wasn't aware that the auto merge was enabled, I will approve your next pr, sorry 😔

np)

**Pull request merged**